on_the_porch/real/app3.py

```
@@ -0,0 +1,903 @@
+    messages.append({"role": "user", "content": user_prompt})
+
+    client = _get_openai_client()
+    try:
+        resp = client.chat.completions.create(
+            model=default_model,
+            messages=messages,
+            temperature=0,
+        )
+        content = resp.choices[0].message.content or ""
+        return _extract_sql_from_text(content)
+    except Exception as exc:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)